config/crd/bases/configuration.konghq.com_kongplugins.yaml

```diff
@@ -41,107 +35,48 @@ spec:
-                  this object represents. Servers may infer this from the endpoint
-                  the client submits requests to. Cannot be updated. In CamelCase.
-                  More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-                type: string
               secretKeyRef:
-                description: SecretValueFromSource represents the source of a secret
-                  value
+                type: object
                 properties:
-                  apiVersion:
-                    description: 'APIVersion defines the versioned schema of this
-                      representation of an object. Servers should convert recognized
-                      schemas to the latest internal value, and may reject unrecognized
-                      values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-                    type: string
                   key:
-                    description: the key containing the value
-                    type: string
-                  kind:
-                    description: 'Kind is a string value representing the REST resource
-                      this object represents. Servers may infer this from the endpoint
-                      the client submits requests to. Cannot be updated. In CamelCase.
-                      More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
                     type: string
                   name:
-                    description: the secret containing the key
                     type: string
-                required:
-                - key
-                - name
-                type: object
             type: object
-          consumerRef:
-            description: ConsumerRef is a reference to a particular consumer
-            type: string
           disabled:
-            description: Disabled set if the plugin is disabled or not
             type: boolean
-          kind:
-            description: 'Kind is a string value representing the REST resource this
-              object represents. Servers may infer this from the endpoint the client
-              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-            type: string
-          metadata:
-            type: object
           plugin:
-            description: PluginName is the name of the plugin to which to apply the
-              config
             type: string
           protocols:
-            description: Protocols configures plugin to run on requests received on
-              specific protocols.
-            enum:
-            - http
-            - https
-            - grpc
-            - grpcs
-            - tcp
-            - tls
-            - udp
             items:
+              enum:
```

Comment:
UDP is missed ?

Reply:
Yes! good catch thank you. Fixed in 8a541a693467d2a8b484a964b07de35a76773944